calculateFrequency.js

```diff
@@ -1,8 +1,6 @@
 // Problem 2: Complete the calculateFrequency function that takes lowercase string as input and returns frequency of all english alphabet. (using only array, no in-built function)
 function calculateFrequency(string) {
```

Comment:
@abansal26 why is this comment not acted upon? And if you have already resolved the issue reply to the comment saying done and preferably the commit in which you corrected it.
All comments addressed to you should be replied to with the appropriate status so that the conversation is complete.